designer/fitting/dwipi.py

```diff
@@ -1469,7 +1469,8 @@ def tensorReorder(self, dwiType):
         KT:         4D image containing KT tensor
         """
         if self.dt is None:
-            assert('Please run dwi.fit() to generate a tensor')
+            assert('Please run dwi.fit() to generate a tensor prior to '
+                   'reordering tensors.')
```

Comment:
You should `raise Exception` instead of assert

Reply:
Fixed in f8e884bff750792a2fcc4ee58a34621b5f19192b